.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         DB_USER: mds
         DB_PASSWORD: mds
       run: |
-        $HOME/.poetry/bin/poetry run pytest --cov=src --cov=migrations/versions --cov-fail-under=96 --cov-report xml
```

Comment:
Is this because of some files it shouldn't be counting or actual lines not being tested? I know this is a high threshold and we've been just decreasing it (I'm also guilty of this), but if there a single simple-ish test that we could write to bring this back up I'd rather do that

Reply:
some lines are not tested because we're not testing 500 errors from indexd, only 401/403/404s

but the main reason is that there is a block in `_create_blank_version` that catches 404s from indexd, which in the current state is not reachable, because the only function that calls `_create_blank_version` catches the 404 earlier. i would rather keep this block for safety though, in case the function is used somewhere else in the future

Reply:
this branch is at exactly 95%